### PR TITLE
Update authors/contributors spec

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: building/tracks/concept-exercises.md [hunk 1]
```diff
@@ -322,18 +317,8 @@ Assume that the user FSharpForever has written an exercise called `basics` for t
...
-    }
-  ],
-  "authors": [
-    {
-      "github_username": "PythonProfessor",
-      "exercism_username": "PythonProfessor"
-    }
-  ],
+  "contributors": ["PythonPerfection"],
+  "authors": ["PythonProfessor"],
```
Comment (iHiD) [inline building/tracks/concept-exercises.md:320]: Could we change this second example so it's more visibly different to the first pls?

**Review by iHiD — APPROVED** [review]

File: building/tracks/concept-exercises.md (hunk 1 above)
Comment (ErikSchierboom) [reply building/tracks/concept-exercises.md:320]: Something like this (pushed change)?

File: building/configlet/lint.md [hunk 2]
```diff
@@ -159,21 +159,15 @@ The `config.json` file should have the following checks:
...
+- The `"authors"` values are treated case-insensitively
 - The `"contributors"` key is optional
-- The `"contributors"` value must be an array
-- The `"contributors[].github_username"` key is required
-- The `"contributors[].github_username"` key must be a non-empty, non-blank string
-- The `"contributors[].github_username"` value is treated case-insensitively
-- The `"contributors[].exercism_username"` key is optional
-- The `"contributors[].exercism_username"` key must be a non-empty, non-blank string
-- The `"contributors[].exercism_username"` value is treated case-insensitively
+- The `"contributors"` value must be a non-empty array
```
Comment (coriolinus) [inline building/configlet/lint.md:167]: Requiring that `contributors` is non-empty is a new requirement in this PR, and it's not obvious that this was intended.

Comment (ErikSchierboom) [reply building/configlet/lint.md:167]: And it should not have been. This is wrong.

Comment (ErikSchierboom) [reply building/configlet/lint.md:167]: Will be fixed in https://github.com/exercism/docs/pull/91/files

**Pull request merged**